index.js

```diff
@@ -315,6 +316,36 @@ slackMessages.action('witnesses', (payload, respond) => {
   };
 });
 
+const peopleCultureChannels = slackChannels.split(',');
+
+cron.schedule('0 22 * * 1', function() {
+  peopleCultureChannels.map(peopleCultureChannel => {
+    let formattedPeopleCultureChannel = peopleCultureChannel.trim();
+
+    sc.groups.info(formattedPeopleCultureChannel)
+      .then(groupDetails => {
+        groupDetails.group.members.map(member => {
+          sc.users.info(member)
+            .then(memberDetails => {
+              if (memberDetails.user.is_bot === false) {
+                if (memberDetails.user.deleted === true) {
+                  // POST to delete? API endpoint
+                }
+
+                axios.post(`${apiBaseUrl}/users`, {
+                  userId: memberDetails.user.id,
+                  email: memberDetails.user.profile.email,
+                  username: memberDetails.user.profile.real_name_normalized,
+                  imageUrl: memberDetails.user.profile.image_48,
+                  location: slackUserLocation(memberDetails.user.tz)
+                }).catch((/* err */) => {});
```

Comment:
Lets have these catch statements log errors to slack. Fix these then I'l merge. We will work on the delete feature after that endpoint is up.

Reply:
The logging errors to Slack is taken care of in the other PR but it can't be merged as of now as @danielwangai is looking into it